assets/prefabs/buildings/Store.prefab

```diff
@@ -0,0 +1,13 @@
+{
+    "GenericBuilding" : {
+     "name" : "Store",
+     "templateNames" : ["HjerlHede:StoreTemplate"],
+     "zone" : "COMMERCIAL",
+     "minSize" : [15, 25],
+     "maxSize" : [15, 25],
+     "isEntity" : true
+    },
+    "SimpleSource": {
+        "needType": "FOOD"
```

Comment:
I'm wondering what other `needTypes` we have so far... a store could be good for all of them...